cltk/phonology/middle_english/transcription.py

```diff
@@ -1,64 +1,87 @@
 
 CONSONANTS = ['b', 'c', 'd', 'f', 'g', 'h', 'l', 'm', 'n', 'p', 'r', 's', 't', 'x', 'ð', 'þ', 'ƿ']
 
+
 class Word:
 
     def __init__(self, word):
         self.word = word
         self.syllabified = ''
 
     def syllabify(self):
+        """
+        Syllabification module for Middle English.
+
+         Throughout the early 11th-14th century, ME went through a process of loss of gemination. Originally, the syllable
+         preceding a geminate was a closed one. The method assumes any occurring geminates will be separated like in Modern
+         English (working both as coda of first syllable and onset of the other.
+
+         Returns:
+             list: string list containing the syllables of the given word
+
+        Examples:
+
+            >>> Word('heldis').syllabify()
+            ['hel', 'dis']
+
+            >>> Word('greef').syllabify()
+            ['greef']
 
-        #Array holding the index of each given syllable
+            Once you syllabify the word, the result will be saved as a class variable
+            >>> word = Word('commaundyd')
+
+            >>> word.syllabify()
+            ['com', 'mau', 'ndyd']
+
+            >>> word.syllabified
+            ['com', 'mau', 'ndyd']
+
+        """
+
+        # Array holding the index of each given syllable
         ind = []
 
         i = 0
-        #Iterate through letters of word searching for the nuclei
+        # Iterate through letters of word searching for the nuclei
         while i < len(self.word) - 1:
 
             if self.word[i] in SHORT_VOWELS:
 
                 nucleus = ''
 
-                #Find cluster of vowels
+                # Find cluster of vowels
                 while self.word[i] in SHORT_VOWELS and i < len(self.word) - 1:
                     nucleus += self.word[i]
                     i += 1
 
                 try:
-                    #Check whether it is suceeded by a geminant
-
-                    # Throught the early 11th-14th century, ME went through a process of loss of gemination (double
-                    # consonants. Originally, the syllable preceding a geminate was a closed one. The following assumes
-                    # any occuring geminates will be separated like in Modern English (working both as coda of first
-                    # syllable and onset of the other
+                    # Check whether it is suceeded by a geminant
 
                     if self.word[i] == self.word[i + 1]:
-                         ind.append(i)
-                         i+=2
-                         continue
+                        ind.append(i)
+                        i += 2
+                        continue
 
-                    #Vowels were shortened before clusters of three consonants
-                    elif sum(c not in CONSONANTS for c in self.word[i:i+3]) == 0:
-                        ind.append(i - 1 if self.word[i:i+3] in TRIPHTHONGS else i)
+                    # Vowels were shortened before clusters of three consonants
```

Comment:
change # to #:
to make this an inline docstring comment.
But maybe this detail could also be promoted to the method level docstring since it sounds like a general rule.